src/step-13-typescript/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "license": "CC-BY-SA-4.0",
+  "name": "typescript-t",
+  "main": "server.ts",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "ts-node-dev --project ./tsconfig.json server.ts",
+    "test": "exit"
```

Comment:
why don't we have tests? let's add them as they exist in the step 9 which we are porting to typescript, especially because tests have interesting typing challenges (sinon stubs)